bugbug/model_calibration.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from sklearn.base import BaseEstimator, ClassifierMixin
+from sklearn.calibration import CalibratedClassifierCV
+from sklearn.metrics import mean_squared_error
+from sklearn.model_selection import train_test_split
+
+
+class IsotonicRegressionCalibrator(BaseEstimator, ClassifierMixin):
+    def __init__(self, base_clf):
+        self.base_clf = base_clf
+        self.calibrated_clf = CalibratedClassifierCV(
+            base_clf, cv="prefit", method="isotonic"
+        )
+
+    def train_test_split(self, X, y, test_size=0.2, random_state=42):
+        X_train, X_test, y_train, y_test = train_test_split(
+            X, y, test_size=test_size, random_state=random_state
+        )
+        return X_train, X_test, y_train, y_test
```

Comment:
will remove it!

Reply:
i noticed that in the model.py file , we do the thing (creating a separate train_test_split method), so I thought it is some kind of convention, but you're right, we dont need a separate method for that!

Reply:
We do that in `model.py` because we need to customize the behaviour for one of the models, which is not the case here (at least for now):

https://github.com/mozilla/bugbug/blob/e7b1ad46f3579f8f1a435be601f2b0f65500084d/bugbug/models/testselect.py#L493-L506

If customizing the behaviour becomes a requirement in future, we then implement a solution (e.g., creating a method for splitting or passing flags/options).